specification/verifiablecredentials/data-plane/Microsoft.VerifiableCredentials/preview/2021-04-05-preview/adminservice.json

```diff
@@ -0,0 +1,241 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2021-04-05-preview",
+    "title": "verifiablecredentials",
+    "x-ms-code-generation-settings": {
+      "name": "verifiablecredentialsClient"
+    }
+  },
+  "host": "beta.did.msidentity.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "bearer_auth": []
+    }
+  ],
+  "securityDefinitions": {
+    "bearer_auth": {
+      "type": "apiKey",
+      "name": "Authorization",
+      "in": "header"
+    }
+  },
+  "paths": {
+    "/{tenantId}/api/portable/v1.0/admin/issuers/{issuerId}": {
+      "put": {
+        "tags": [
+          "Issuers"
+        ],
+        "operationId": "AdminService_IssuerPut",
```

Comment:
For operations, we try to avoid REST-specific nomenclature.  How about `AdminService_UpdateIssuer`?

Reply:
Updated